### DISCUSSION TITLE
How to disable ssl for the website.

### DISCUSSION THREAD

**Discussion by NavaneethVijay** [markshust/docker-magento#473] [Q&A] [answered]
Even when then secure and unsecure base urls are set with http and I've disable usesecure url in admin and frontend, I'm still seeing that the website is redirected to https and shows too many redirects error.
I do not want to use ssl or use secure urls in my development environment due to unverified ssl error. 
What is the best way to remove this option.

**Comment by markshust** [comment, marked as answer]
SSL is required, no way to disable it. If you are having issues with the cert, you should be able to bypass the "self-signed cert" or add it in the SSL registry on your computer.
